packages/hub/queries/email-card-drop-requests.ts

```diff
@@ -11,15 +11,21 @@ interface EmailCardDropRequestsQueriesFilter {
   verificationCode?: string;
 }
 
+const IS_NOT_OLD = `requested_at > now() - interval '60 minutes'`;
+const IS_OLD = `requested_at <= now() - interval '60 minutes'`;
```

Comment:
I think this would be nice to config-ify so it can be used elsewhere without duplication elsewhere, like in the text of the verification email. `HUB_EMAIL_CARD_DROP_RATE_LIMIT_PERIOD_MINUTES` is an admittedly clunky name but it’s a pattern I’ve seen to reduce ambiguity. I don’t think this needs to be an environment variable though.

Reply:
In that vein I added `expiryMinutes` [here](https://github.com/cardstack/cardstack/pull/2971/files#diff-b38fc6a0079f557fa66aa5471a73510a2020d1423c74f28cc9f3f318b64ae823R87) 🤓